core/v2/types.go

```diff
@@ -413,9 +218,11 @@ type Attestation struct {
 }
 
 type BlobVerificationInfo struct {
-	BlobCertificate *BlobCertificate
-	BlobIndex       uint32
-	InclusionProof  []byte
+	*BatchHeader
```

Comment:
We had this in the design doc: 

```
message SignedBatch {
	BatchHeader batch_header = 1;
  Attestation non_signer_stakes_and_signature = 2;
}

message BlobVerificationInfo {
  BlobCertificate blob_certificate = 1;
  uint32 blob_index = 2;
  bytes inclusion_proof = 3;
}
```

Is there an update here?

Reply:
Yeah protobuf interface isn't changed. Just internal structs. 
In fact, I was thinking about removing `BlobHeader` from `BlobVerificationInfo` as the cert can be looked up by the blobKey. What do you think?

Reply:
>In fact, I was thinking about removing BlobHeader from BlobVerificationInfo as the cert can be looked up by the blobKey. What do you think?
Removing from the internal struct or the interface? It looks like you already removed it from the struct? That seems fine to me, but haven't yet thought through exactly how this is going to be used...

Reply:
Oops. Right I already did lol 
From controllers perspective, this will be used to look up attestation given some blob.